lib/pnode/message.ml

```diff
@@ -20,6 +20,7 @@ type t = {
   sender : Address.t;
   recipients : Address.t list;
   payload : bytes;
+  payload_signature_opt : bytes option;
```

Comment:
No need for opt in the name since the type is clear about this IMO.

Reply:
You're right, done.